### PR TITLE
Paalman pings documentation

### PR DESCRIPTION
**Pull request by peterfpeterson**
The workflow diagrams are correct, but I found them much more confusing than the original equation from the paper.

**To test:**

Build the sphinx documentation and see if things make more sense.

*There is no associated issue.*

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/algorithms/ApplyPaalmanPingsCorrection-v1.rst [hunk 1]
```diff
@@ -9,18 +9,64 @@
...
-
-This algorithm can be used to apply absorption corrections calculated with
-either the :ref:`algm-CylinderPaalmanPingsCorrection` and
-:ref:`algm-FlatPlatePaalmanPingsCorrection` algorithms as well as the legacy
-indirect calculate corrections routine, providing that the sample and container
-are first converted to wavelength and the corrections are interpolated to match
-the sample as demonstrated in the example below.
+Applies absorption corrections calculated in the Paalman and Pings
+absorption factor format. The various partial absorption factors are
+denoted as :math:`A_{a,b}` where the subscript :math:`b` denotes the
```
Comment (AndreiSavici) [inline docs/source/algorithms/ApplyPaalmanPingsCorrection-v1.rst]: Subscript `b` and subscript `b`, so where is `a`?

**Comment by SpencerHowells** [comment]
In the definition of the terms like As,sc in the column headed 'absorbed by' the phrase 'sample or container' should be 'sample and container'.

**Comment by peterfpeterson** [comment]
The neutron can only be absorbed once. Therefore I thought "or" was the more appropriate term.

**Review by AndreiSavici — APPROVED** [review]
Looks OK now

**Pull request merged**